libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -102,6 +102,11 @@
 		82D4642519C7C8170006BDFE /* Foundation.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 8220B74F16D804CA00EC3921 /* Foundation.framework */; };
 		82D4642619C7C8170006BDFE /* CoreGraphics.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 8280EBCC16E14F7000768DE8 /* CoreGraphics.framework */; };
 		82D4642719C7C8170006BDFE /* UIKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 8280EBB516E14E9F00768DE8 /* UIKit.framework */; };
+		93E423A12162993D00274DF3 /* SFSDKSafeMutableArray.m in Sources */ = {isa = PBXBuildFile; fileRef = 93E423A02162993D00274DF3 /* SFSDKSafeMutableArray.m */; };
+		93E423A32162B85100274DF3 /* SFSDKSafeMutableArrayTests.m in Sources */ = {isa = PBXBuildFile; fileRef = 93E423A22162B85100274DF3 /* SFSDKSafeMutableArrayTests.m */; };
+		93E423A42162DAF200274DF3 /* SFSDKSafeMutableArray.h in Headers */ = {isa = PBXBuildFile; fileRef = 93E4238C216297BA00274DF3 /* SFSDKSafeMutableArray.h */; settings = {ATTRIBUTES = (Public, ); }; };
+		93E423A52162DB0E00274DF3 /* SFSDKSafeMutableArray.h in Headers */ = {isa = PBXBuildFile; fileRef = 93E4238C216297BA00274DF3 /* SFSDKSafeMutableArray.h */; settings = {ATTRIBUTES = (Public, ); }; };
```

Comment:
Duplicate entry here?

Reply:
Make sure it's added to both targets (static and dynamic framework).

Reply:
I added the file to both targets. 
![screen shot 2018-10-02 at 10 32 03 am](https://user-images.githubusercontent.com/3354167/46365860-80699b00-c62e-11e8-9a0d-43462ee2c521.png)